ui/README.md

```diff
@@ -45,6 +45,21 @@ accept the invalid certificate warning in your browser for this to work.
 - `ember serve local`
 - The app will be available at [http://localhost:4200](http://localhost:4200).
 
+If you need to build the server and run it locally, you'll want to stop the existing instance, build and reinstall it in docker:
+
+- `docker stop waypoint-server; docker rm waypoint-server; docker volume prune -f`
+- `make docker/evanphx`
```

Comment:
Seeing this reminds me that maybe we should have a generic make target for building the server? Maybe we could rename this to something like `make docker/server` or something in a separate PR? @evanphx @krantzinator  @mitchellh 

I've also been using the `docker/mitchellh` target because it tags the server as `dev` rather than overriding `latest`

Reply:
I'm happy to open that PR now, maybe I'll just copy what I've been using and name it `docker/server`.

Reply:
Thanks @briancain. I just reworded this

Reply:
Thanks! :) Just merged that one so I think this should be good to go